CONTRIBv2.md

```diff
@@ -28,7 +28,7 @@ Interested in writing a story with the Maroon Data Team? Follow the steps below.
 4. Download node which will allow you to run Javascript, the programming language we use to write our visualizations in:
     - [node.js](https://nodejs.org/en) `v20.9.0`
         - To install the correct version, open the terminal and run:
-            1. Install [Node Version Manager](https://github.com/nvm-sh/nvm) with `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash` 
+            1. Install [Node Version Manager](https://github.com/nvm-sh/nvm) with `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash` . If you are getting error of alias contains slash (likely if you are on Windows) try using `nvm-setup.exe` from the [Windows release site](https://github.com/coreybutler/nvm-windows/releases).
```

Comment:
`If you are getting error of alias contains slash` -> `If you are receiving the error "alias contains slash"`

Something like that.